frontend/src/pages/legacy/en-US.md

```diff
@@ -0,0 +1,17 @@
+## Legacy
```

Comment:
Oops, one more bug here: This says "Legacy", but the integration test checks for it saying "Legacy Test Pilot", so the integration test fails. Change this to say "Legacy Test Pilot".

Reply:
Got it ! - Thank you for pointing that out